tldw_Server_API/tests/Character_Chat_NEW/property/test_character_properties.py

```
@@ -543,7 +543,7 @@ def initialize_manager(self):
         """Initialize the manager if not already done."""
         if self.manager is None:
             import tempfile
-            self.db_path = tempfile.mktemp(suffix='.db')
+            self.db_path = tempfile.mktemp(suffix='.db')  # nosec B306
```

Ruff (0.15.1)

[error] Use of insecure and deprecated function (`mktemp`)

(S306)